Containerfile

```diff
@@ -1,8 +1,15 @@
 FROM fedora
 WORKDIR /app
 
-RUN sudo dnf install -y jq units stress stress-ng procps-ng
-RUN  curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod a+x /usr/local/bin/kubectl
+ENV OC_URL=https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
```

Comment:
Do we need this?

If not, then I'd propose to drop it.

Reply:
@fabiand You're right! Dropped.